README.md

```diff
@@ -13,14 +13,18 @@
   - [Current Status](#current-status)
 - [Room For Improvement](#room-for-improvement)
 - [Acknowledgements](#acknowledgements)
+___
 
+![GitHub](https://img.shields.io/github/last-commit/gschool-blue-ocean/admissions-hub-20/main)
+
+___
 ## General Information
 
-This project is the culmination of months of hard work growing our skills within the Galvanize course. We came together as a team to rework an existing project from a previous cohort in order to meet the demands of a client.
+This project is the culmination of months of **🚀 hard work 🚀** growing our skills within the Galvanize course. We came together as a team to rework an existing project from a previous cohort in order to meet the demands of a client.
 
-- Project Manager: Chris Stewart (pri), Rane Gray (alt)
-- UI Manager: Rane Gray (pri), Daniel Purdy (alt)
-- Architecture Manager: Joe Goralczyk (pri), Jordan Carlson (alt)
+-  👨🏻‍✈️ 📝 Project Manager: Chris Stewart (pri), Rane Gray (alt)
+-  👨🏻‍🎨 🖼️ UI Manager: Rane Gray (pri), Daniel Purdy (alt)
+-  👷🏻‍♂️ 🏗️ Architecture Manager: Joe Goralczyk (pri), Jordan Carlson (alt)
 
 Working together as an Agile group, we worked hard to redesign this application visually and functionally per our client's needs. We've reworked the authentication system that correctly stores hashed passwords in the database and keeps the user logged in, a dashboard with an interactive table element that allows for tracking prospective stuents, and an interview page that implements a usable coding space to collaborate between admissions personnel and students.
```

Comment:
I think we should include that we use web tokens, it seemed like a real achievement to get them working. Maybe "we refactored authentication to add protected routes and json web tokens along with hashed password storage" and with the shared coding space include we use websockets "a usable coding space to collaborate between admissions personnel and students using an express websocket server and sandpack code editors". Adding those technical details to give a little more weight to the amount of understanding that stuff requires.

Reply:
Done